build/Build.cs

```diff
@@ -219,22 +219,43 @@ public Build()
                                                    .Where(project => MigratedCalamariFlavours.Flavours.Contains(project.Name)
                                                                      || migratedCalamariFlavoursTests.Contains(project.Name));
 
-                    var packagesToBuild = calamariFlavours
-                        .SelectMany(project => project.GetTargetFrameworks(), (p, f) => new
-                        {
-                            Project = p,
-                            Framework = f,
-                            CrossPlatform = IsCrossPlatform(f),
-                        })
-                        .SelectMany(packageToBuild => packageToBuild.Project.GetRuntimeIdentifiers(), (packageToBuild, runtimeIdentifier) => new
-                        {
-                            packageToBuild.Project,
-                            packageToBuild.Framework,
-                            Architecture = packageToBuild.CrossPlatform ? runtimeIdentifier : null,
-                            packageToBuild.CrossPlatform
-                        })
-                        .Distinct(t => new { t.Project.Name, t.Architecture, t.Framework });
+                    // for NetFx target frameworks, we use "netfx" as the architecture, and ignore defined runtime identifiers
+                    var netFxPackages = calamariFlavours
```

Comment:
Can we pull the below snippet into it's own variable and share between the netfx and cross-platform packages?
`calamariFlavours
                      .SelectMany(project => project.GetTargetFrameworks()!, (p, f) => new
                      {
                          Project = p,
                          Framework = f,
                          CrossPlatform = IsCrossPlatform(f)
                      })`